index.html

```diff
@@ -32,9 +44,23 @@
-        <h2 class="headingstyle">Sub Title</h2>
-        <p>I hate cucumber pls dont throw it at me hopped up on catnip spill litter box, scratch at owner, destroy all furniture, especially couch meeeeouw. Poop in a handbag look delicious and drink the soapy mopping up water then puke giant foamy fur-balls then cats take over the world. Get scared by sudden appearance of cucumber get scared by doggo also cucumerro catching very fast laser pointer or prance along on top of the garden fence, annoy the neighbor's dog and make it bark. Climb a tree, wait for a fireman jump to fireman then scratch his face pet right here, no not there, here, no fool, right here that other cat smells funny you should really give me all the treats because i smell the best and omg you finally got the right spot and i love you right now scratch i heard this rumor where the humans are our owners, pfft, what do they know?!. Eat owner's food friends are not food or small kitty warm kitty little balls of fur cats are the world hey! you there, with the hands experiences short bursts of poo-phoria after going to the loo yet dream about hunting birds. Cat milk copy park pee walk owner escape bored tired cage droppings sick vet vomit licks your face. Fight an alligator and win cat meoooow i iz master of hoomaan, not hoomaan master of i, oooh damn dat dog give me some of your food give me some of your food give me some of your food meh, i don't want it hiding behind the couch until lured out by a feathery toy but wack the mini furry mouse skid on floor, crash into wall . Terrorize the hundred-and-twenty-pound rottweiler and steal his bed, not sorry ooh, are those your $250 dollar sandals? lemme use that as my litter box and annoy the old grumpy cat, start a fight and then retreat to wash when i lose so destroy couch as revenge. Fall asleep on the washing machine.</p>
-    </body>
-</html>
\ No newline at end of file
+        <h1 class="headingstyle">What is a Hackathon?</h1>
```

Comment:
It would be better if we did not use class or id tags if the an element can share a style. This way you can remove the class attritube in the p and the o elements. Please feel free to have me explain further if this is not clear.

I initially used the class headingstyle so that we wouldn't have to be redundant in saying h1, h2, h2, h3, etc in the style sheet.

Reply:
Actually, I think we can even take out the class headingstyles I initially put in and just use h1 and h2 in the style section since those are the only heading types we'll be using.